cmd/kaniko-acr/main.go

```diff
@@ -368,7 +381,11 @@ func fetchACRToken(tenantId, token, registry string) (string, error) {
 }
 
 func setupACRCert(cert string) error {
-	err := ioutil.WriteFile(ACRCertPath, []byte(cert), 0644)
+	decoded, err := base64.StdEncoding.DecodeString(cert)
```

Comment:
Add a comment in the flag that it is expected that cert needs to be in base64 format as an input.

Reply:
done